library/registry

```diff
@@ -2,7 +2,11 @@ Maintainers: Milos Gajdos (@milosgajdos),
              Sebastiaan van Stijn (@thajeztah)
 GitRepo: https://github.com/docker/distribution-library-image.git
 GitFetch: refs/heads/master
-GitCommit: 39dd72feaab7066334829d6945c54bc51a0aee98
+
+Tags: 3.0.0-alpha.1
```

Comment:
@tianon should this has `latest` tag instead of keeping it with the `v2` release? I think it should 🤔 cc @thaJeztah

Reply:
Depends on what you want users to "use by default" if they don't know any better -- putting that on an Alpha seems a bit ambitious to me, but you're in a better position to know whether 3.0.0-alpha.1 is suitable for general use than I am. :smile:

Reply:
(in other words, "latest" actually means "default", not literally "the newest one")

Reply:
Let me chat to @thaJeztah first 😄 my thinking is: if we don't mark it as `latest` it will have less of a chance to get properly tested; folks can always pick an explicit `v2` tag if they wish to do so. BRB, don't merge yet please 😄

Reply:
We should probably wait until GA, I don’t think we usually ship pre-releases as “latest”.

Pre-releases are for testing, but should be opt-in

Reply:
ok, @tianon, we're sticking to your original suggestion 😄 `latest` = latest stable which is still `2.x`. Merge away please 🙇‍♂️